Demo Apps/Podcasts/Podcasts/PDCAppDelegate.m

```diff
@@ -38,8 +38,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 		// Creating the view controller with features.
 		MTZWhatsNewGridViewController *vc = [[MTZWhatsNewGridViewController alloc] initWithFeatures:whatsNew];
 		// Customizing the background gradient.
-		vc.backgroundGradientTopColor = [UIColor colorWithHue:0.77 saturation:0.77 brightness:0.76 alpha:1];
-		vc.backgroundGradientBottomColor = [UIColor colorWithHue:0.78 saturation:0.6 brightness:0.95 alpha:1];
+        vc.backgroundGradientTopColor = [UIColor colorWithHue:0.77 saturation:0.77 brightness:0.76 alpha:1];
```

Comment:
It seems this whitespace change was accidentally committed.

Reply:
Yes, sorry. Saw too late, though :/

Reply:
Should I do reverse commit for those two lines?